src/Http/Session.php

```diff
@@ -604,15 +604,25 @@ public function renew()
 
         $this->start();
         $params = session_get_cookie_params();
-        setcookie(
-            session_name(),
-            '',
-            time() - 42000,
-            $params['path'],
-            $params['domain'],
-            $params['secure'],
-            $params['httponly']
-        );
+        if (PHP_VERSION_ID >= 70300) {
```

Comment:
I wonder if we should use https://www.php.net/manual/en/function.session-set-cookie-params.php somewhere to actually set the samesite option. Since we now only 'copy' it and never set it.

But perhaps we leave that to a new PR?

Reply:
This particular `setcookie()` call is deleting previous cookie when renewing session, I doubt `session_set_cookie_params()` works for that case, plus as the doc mentions it needs to called before `session_start()` is called which isn't the case either here.

Reply:
Sorry I meant in the `start` method. But it seems we can use `defaults.session.cookie_samesite` for that in the config.

Reply:
Just tested, this:

```php
    'Session'        => [
        'defaults'   => 'php',
        'ini' => [
            'session.cookie_samesite' => 'None',
        ],
    ],
```

indeed works for the session cookie :)